example/lib/ios_popover/_ios_toolbar.dart

```diff
@@ -0,0 +1,594 @@
+      ..autoPaginated = _autoPaginated
+      ..pages = _menuPagesToPageInfo();
+  }
+
+  List<_MenuPageInfo>? _menuPagesToPageInfo() {
+    if (_pages == null) {
+      return null;
+    }
+
+    final pageInfoList = <_MenuPageInfo>[];
+
+    // Starts from 1 because index 0 is the previous button.
+    int currentIndex = 1;
+    for (final page in _pages!) {
+      int endingIndex = currentIndex + page.items.length;
+      pageInfoList.add(
+        _MenuPageInfo(
+          startingIndex: currentIndex,
+          endingIndex: endingIndex,
+        ),
+      );
+      currentIndex = endingIndex;
+    }
+
+    return pageInfoList;
+  }
+}
+
+class _IosPagerParentData extends ContainerBoxParentData<RenderBox> {}
+
+/// Render a paginated menu.
+class _RenderIosPagedMenu extends RenderBox
+    with
+        ContainerRenderObjectMixin<RenderBox, _IosPagerParentData>,
+        RenderBoxContainerDefaultsMixin<RenderBox, _IosPagerParentData> {
+  _RenderIosPagedMenu({
+    required _MenuPageController controller,
+    List<_MenuPageInfo>? pages,
+    bool autoPaginated = true,
+  })  : _controller = controller,
+        _pages = pages,
+        _autoPaginated = autoPaginated;
+
+  /// [Paint] used to paint the lines between items.
+  final _linePaint = Paint()..color = const Color(0xFF555555);
+
+  _MenuPageController _controller;
+  _MenuPageController get controller => _controller;
+  set controller(_MenuPageController value) {
+    if (_controller != value) {
+      _controller = value;
+      markNeedsLayout();
+    }
+  }
+
+  bool _autoPaginated;
+  bool get autoPaginated => _autoPaginated;
+  set autoPaginated(bool value) {
+    if (_autoPaginated != value) {
+      _autoPaginated = value;
+      markNeedsLayout();
+    }
+  }
+
+  List<_MenuPageInfo>? _pages;
+  List<_MenuPageInfo>? get pages => _pages;
+  set pages(List<_MenuPageInfo>? value) {
+    if (_pages != value) {
+      _pages = value;
+      markNeedsLayout();
+    }
+  }
+
+  @override
+  void attach(covariant PipelineOwner owner) {
+    super.attach(owner);
+    _controller.addListener(markNeedsLayout);
+  }
+
+  @override
+  void detach() {
+    _controller.removeListener(markNeedsLayout);
+    super.detach();
+  }
+
+  @override
+  void setupParentData(covariant RenderObject child) {
+    if (child.parentData is! _IosPagerParentData) {
+      child.parentData = _IosPagerParentData();
+    }
+  }
+
+  @override
+  void performLayout() {
+    if (autoPaginated) {
+      _computePages();
+    }
+    _scheduleUpdateControllerPageCount();
+
+    final hasMultiplePages = _pages!.length > 1;
+
+    // Children include the navigation buttons.
+    final children = getChildrenAsList();
+
+    double height = 0;
+    double width = 0;
+
+    // Layout all the children and get the maxHeight.
+    for (int i = 0; i < children.length; i++) {
+      final child = children[i];
+      child.layout(constraints, parentUsesSize: true);
+      height = max(height, child.size.height);
```

Comment:
Shouldn't we be imposing the height on all the children? I assumed the iOS toolbar had a constant height for all circumstances...

Reply:
Update to configure a fixed height.